packages/logger/src/index.ts

```diff
@@ -86,16 +44,23 @@ export const logger: ILogger = {
 };
 
 // tslint:disable-next-line: variable-name
-export const NSlogger = (namespace: string = ''): ILogger => ({
-  info: instance.info.bind(instance),
-  warn: instance.warn.bind(instance),
-  debug: LOG_LEVEL === 'debug' ? getDebugger(namespace) : () => {},
-  error: instance.error.bind(instance),
-});
+export const NSlogger = (namespace: string = ''): ILogger => {
+  instance.defaultMeta = {
+    ...instance.defaultMeta,
+    namespace,
+  };
+
+  return {
+    info: instance.info.bind(instance),
+    warn: instance.warn.bind(instance),
+    debug: ENV.logLevel === 'debug' ? getDebugger(namespace) : () => {},
+    error: instance.error.bind(instance),
+  };
+};
 
 export interface ILogger {
-  error: (message: Error | string, ...args: any[]) => void;
-  warn: (message: Error | string, ...args: any[]) => void;
+  error: (message: string, ...args: any[]) => void;
+  warn: (message: string, ...args: any[]) => void;
```

Comment:
Let me know if you agree with this change or not. But, it makes it more consistent with `TransformableInfo`, other logger functions, and makes it easier to convert this into a GCP Error Reporting format

Reply:
No specific issues with it so 👌